### PR TITLE
feat: expose base driver version via inherited drivers

### PR DESCRIPTION
**Pull request by jlipps**
This way we can always determine basedriver compatibility between appium server and drivers

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: lib/basedriver/driver.js [hunk 1]
```diff
@@ -30,6 +32,12 @@ const ON_UNEXPECTED_SHUTDOWN_EVENT = 'onUnexpectedShutdown';
 
 class BaseDriver extends Protocol {
 
+  /**
+   * Make the basedriver version available so for any driver which inherits from this package, we
+   * know which version of basedriver it inherited from
+   */
+  static baseVersion = BASEDRIVER_VER;
```
Comment (mykola-mokhnach) [inline lib/basedriver/driver.js:39]: is `static` supported in ECMA standard that Appium uses?

File: lib/basedriver/driver.js [hunk 2]
```diff
@@ -20,6 +21,7 @@ B.config({
   cancellation: true,
 });
 
+const BASEDRIVER_VER = require(path.resolve(__dirname, '..', '..', '..', 'package.json')).version;
```
Comment (mykola-mokhnach) [inline lib/basedriver/driver.js:24]: chromedrivers uses a bit different approach: https://github.com/appium/appium-chromedriver/blob/78b0ed6ae4d973783733eb626ae14b5b2edc716c/lib/utils.js#L9

which one is right?

File: lib/basedriver/driver.js (hunk 1 above)
Comment (jlipps) [reply lib/basedriver/driver.js:39]: it was already in the list of our babel transpilation plugins. i merely had to add it to our linter as well.

File: lib/basedriver/driver.js (hunk 2 above)
Comment (jlipps) [reply lib/basedriver/driver.js:24]: right now i'm assuming that everything is run transpiled. if we're ever in a scenario where that's not the case, the test will tell us.

Comment (jlipps) [reply lib/basedriver/driver.js:24]: I think chromedriver cares because the install script needs to run untranspiled

**Comment by jlipps** [comment]
published in 8.0.0-beta.3

**Pull request merged**